src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -5156,8 +5156,11 @@ public partial class IndentedTextWriter : System.IO.TextWriter
         [System.Diagnostics.CodeAnalysis.AllowNullAttribute]
         public override string NewLine { get { throw null; } set { } }
         public override void Close() { }
+        public override System.Threading.Tasks.ValueTask DisposeAsync() => throw null;
```

Comment:
Here and elsewhere: I'm not sure if using `=>` would cause trouble, so let's stick to safety and use the same pattern used in all our refs (traditional parenthesis):
```suggestion
        public override System.Threading.Tasks.ValueTask DisposeAsync() { throw null; }
```

Reply:
Fixed